README.md

```diff
@@ -1,39 +1,58 @@
-# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=master)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
-Kubernetes deployment of RADAR-base.
+# RADAR-Kubernetes [![Build Status](https://travis-ci.org/RADAR-base/RADAR-Kubernetes.svg?branch=dev)](https://travis-ci.org/RADAR-base/RADAR-Kubernetes)
+The repository contains Helm charts and other configuration needed to install RADAR-base on a Kubernetes cluster.
 
+## Prerequisites
+#### Kubernetes cluster 
+You need to have a working Kubernetes installation and there are 3 ways to have that:
+* From cloud providers such as: [AWS](https://aws.amazon.com/eks/), [GCP](https://cloud.google.com/kubernetes-engine/), [Azure](https://docs.microsoft.com/en-us/azure/aks/), etc
+* Install it on your own servers with: [Rancher](https://rancher.com/), [Kubespray](https://github.com/kubernetes-sigs/kubespray), [Kubeadm](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/), [OpenShift](https://www.okd.io/), etc
+* Install it on your local machine with: [Minikube](https://kubernetes.io/docs/setup/learning-environment/minikube/), [K3S](https://k3s.io/), etc
```

Comment:
Could we add a sample server spec for local machine setup? @K1Hyve

Reply:
According to @blootsvoets a full installation with logging and monitoring would need 8 CPU cores and 32 GB of RAM 
But the minimum would be 4 CPU cores and 16 GB of memory.